orm/CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
-### API-Breaking Changes
+### API Breaking Changes
 
-- [14822](https://github.com/cosmos/cosmos-sdk/pull/14822) Migrate to cosmossdk.io/core genesis API
\ No newline at end of file
+- [14822](https://github.com/cosmos/cosmos-sdk/pull/14822) Migrate to cosmossdk.io/core genesis API
+
+### State-machine Breaking Changes
```

Comment:
I think we should have docs, upgrading tooling on that. Otherwise, it feels a bit like "good luck".

Reply:
I'm planning to push another PR for ORM migrations that will automate this